activestorage/test/controllers/representations/proxy_controller_test.rb

```diff
@@ -37,6 +38,20 @@ class ActiveStorage::Representations::ProxyControllerWithVariantsTest < ActionDi
     assert_equal 67, image.height
   end
 
+  test "showing untracked variant" do
+    without_variant_tracking do
+      get rails_blob_representation_proxy_url(
+        disposition: :attachment,
+        filename: @blob.filename,
+        signed_blob_id: @blob.signed_id,
+        variation_key: ActiveStorage::Variation.encode(@transformations))
+
+      assert_response :ok
+      assert_match(/^attachment/, response.headers["Content-Disposition"])
+      assert_equal @blob.representation(@transformations).download, response.body
```

Comment:
If you agree with this test: 

then I would like to change all 
```ruby
image = read_image(@blob.variant(resize_to_limit: [100, 100]))
    assert_equal 100, image.width
    assert_equal 67, image.height
```
to 
```ruby
assert_equal @blob.representation(@transformations).download, response.body
```
Because the `read_image`, in my opinion, does not test anything related to ProxyController GET request + these tests are redundant with other tests.

And, If you agree with this applied suggestion, would you like a separate PR or can i change it inside this one plese.

Reply:
I share your confusion from https://github.com/rails/rails/pull/50098#discussion_r1398278023 — I am unclear on why the other tests were written that way.  But let's stay focused on fixing non-tracked variants in this PR.  Then we can investigate the other tests further in a follow-up PR.

As for this particular assertion, it appears to be the correct approach.

Reply:
Alright let's do this in a follow up PR 👍🏼